package.json

```diff
@@ -1,54 +1,71 @@
-        "@ordermentum/eslint-config-ordermentum": "^2.4.0",
-        "@typescript-eslint/eslint-plugin": "5.62.0",
-        "@typescript-eslint/parser": "5.62.0",
-        "eslint": "8.57.1",
-        "eslint-config-airbnb": "19.0.4",
-        "eslint-config-airbnb-base": "15.0.0",
-        "eslint-config-prettier": "8.10.0",
-        "eslint-plugin-import": "2.31.0",
-        "eslint-plugin-jsx-a11y": "6.10.2",
-        "eslint-plugin-mocha": "10.5.0",
-        "eslint-plugin-prettier": "4.2.1",
-        "eslint-plugin-promise": "^6.2.0",
-        "eslint-plugin-unicorn": "45.0.2",
-        "mocha": "10.2.0",
-        "turbo": "1.11.2",
-        "typescript": "5.1.3"
-    },
-    "scripts": {
-        "build": "yarn run turbo run build --filter=steveo --filter=scheduler-prisma --filter=scheduler-sequelize &&  yarn run turbo run build --filter=!steveo --parallel",
-        "lint": "yarn run turbo run lint --parallel",
-        "test": "yarn run turbo run test",
-        "release": "yarn build && changeset version && changeset publish"
-    },
-    "workspaces": [
-        "packages/*"
-    ],
-    "packageManager": "yarn@1.22.22",
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/ordermentum/steveo.git"
-    },
-    "author": "",
-    "private": true,
-    "license": "Apache-2.0",
-    "bugs": {
-        "url": "https://github.com/ordermentum/steveo/issues"
-    },
-    "homepage": "https://github.com/ordermentum/steveo#readme"
+  "name": "steveo",
+  "version": "1.0.0",
+  "description": "Steveo Monorepo",
+  "directories": {
+    "doc": "docs"
+  },
+  "dependencies": {
+    "debug": "4.4.0",
+    "mocha": "10.2.0",
+    "sinon": "^15.2.0",
+    "ts-node": "10.9.2"
+  },
+  "devDependencies": {
+    "@changesets/cli": "^2.26.2",
+    "@ordermentum/eslint-config-ordermentum": "^3.0.1",
+    "@typescript-eslint/eslint-plugin": "^8.0.0",
+    "@typescript-eslint/parser": "^8.0.0",
+    "eslint": "^8.57.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-config-turbo": "^2.4.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jsx-a11y": "^6.7.1",
+    "eslint-plugin-jsx-no-leaked-values": "^0.1.24",
+    "eslint-plugin-mocha": "^10.0.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-promise": "^7.0.0",
+    "eslint-plugin-react": "^7.32.1",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-tailwindcss": "^3.17.5",
+    "eslint-plugin-unicorn": "^56.0.0",
+    "mocha": "10.2.0",
+    "prettier": "^2.8.3",
+    "turbo": "^2.4.0",
+    "typescript": "5.1.3"
+  },
+  "scripts": {
+    "build": "yarn run turbo run build --filter=steveo --filter=scheduler-prisma --filter=scheduler-sequelize &&  yarn run turbo run build --filter=!steveo --parallel",
+    "lint": "yarn run turbo run lint --parallel",
+    "lint:test": "yarn run turbo run lint --parallel",
```

Comment:
`lint` and `lint:test` are the same command - did you mean to add duplicates?

Reply:
ah, no. 
I was meaning to add it and then realised it isn't worth adding.